tfm_ns_import.yaml

```diff
@@ -152,16 +146,6 @@
                 "dst": "tfm/targets/TARGET_ARM_SSG/TARGET_MUSCA_B1/device/tfm_peripherals_def.h"
             }
          ],
-         "ARM_MUSCA_A1": [
-            {
-                "src": "platform/ext/target/musca_a/Device/Include/platform_base_address.h",
-                "dst": "tfm/targets/TARGET_ARM_SSG/TARGET_MUSCA_A1/device/platform_base_address.h"
-            },
-            {
-                "src": "platform/ext/target/musca_a/tfm_peripherals_def.h",
-                "dst": "tfm/targets/TARGET_ARM_SSG/TARGET_MUSCA_A1/device/tfm_peripherals_def.h"
```

Comment:
These destination folders (inside this regression test repo) are still present and should be removed as part of this commit.

`./tfm/targets/TARGET_ARM_SSG/TARGET_MUSCA_A1`